voc/python/ast.py

```diff
@@ -1894,6 +1917,16 @@ def compare_to(arg):
 
     @node_visitor
     def visit_Call(self, node):
+        # Check if the callable function contains 
+        # not implemented features, if it does, abort
+        # compilation
+        if node.func.id in self.notimplemented:
+            error = self.notimplemented[node.func.id]
```

Comment:
To avoid name clashing (e.g. functions in different modules/classes with the same name), it's best to store in this dictionary something that identify the specific node being compiled.
My suggestion is to use `'function-{}-{}'.format(node.func.id, id(node))`.